Controller/Checkout/Cart.php

```diff
@@ -41,7 +43,11 @@ public function execute()
         }
 
         $redirect = $this->resultRedirectFactory->create();
-        $redirect->setPath('checkout/cart');
+        if (sizeof($params) > 0) {
+            $redirect->setPath('checkout/cart',['_query'=>$params]);
```

Comment:
```suggestion
            $redirect->setPath('checkout/cart', ['_query'=>$params]);
```

Reply:
what happens if params is null/has no params and is passed into this as the second arg?

Reply:
good point -- should I change to if ($params != NULL && sizeof($params) > 0)  ?

Reply:
no i'm asking what happens if params is null and get's passed into the setPath function, is the if/else necessary?

Reply:
Ohhh I get what you're saying now

Reply:
@kamil-klasicki what gets returned by "['_query'=>$params]" if $params is null? Also why are we doing "['_query'=>$params]" instead of just passing $params?